src/octoprint/templates/sidebar/state.jinja2

```diff
@@ -11,7 +11,7 @@
 <span title="{{ _('Current timelapse configuration')|edq }}">{{ _('Timelapse') }}</span>: <strong data-bind="text: timelapseString"></strong><br>
 <!-- /ko -->
 <!-- ko foreach: filament -->
-<span data-bind="text: 'Filament (' + name() + '): ', title: 'Filament usage for ' + name()"></span><strong data-bind="text: formatFilament(data())"></strong><br>
+<span data-bind="text: '{{ _('Filament')|edq }} (' + name() + '): ', title: 'Filament usage for ' + name()"></span><strong data-bind="text: formatFilament(data())"></strong><br>
```

Comment:
This will fail if the translation for whatever reason contains single quotes (it should never do this, but still, this code is wrong this way).

Given that we have a bit of a longer text here with some placeholders, it would probably make more sense to turn this into a formatting string with actual placeholders...

Also, "Filament usage for ..." is still not translatable, should probably also get adjusted as its user visible as well.

Reply:
Furthermore, I didn't see any tooltip or hint for "Filament usage for..." on the page when I hovered my mouse over the content.

Reply:
> Furthermore, I didn't see any tooltip or hint for "Filament usage for..." on the page when I hovered my mouse over the content.

Indeed, that's a bug in the current code 😅 Sets `title: ...`, should be setting `attr: {title: ...}`.

I'll take a look at this